apps/dashboard/src/main/java/com/akto/action/LoginAction.java

```diff
@@ -190,6 +191,22 @@ public static String loginUser(User user, HttpServletResponse servletResponse, b
                                 Updates.set(User.LAST_LOGIN_TS, Context.now())
                         )
                 );
+                /*
+                 * Creating datatype to template on user login.
+                 * TODO: Remove this job once templates for majority users are created.
+                 */
+                service.submit(() -> {
+                    try {
+                        for (String accountIdStr : user.getAccounts().keySet()) {
+                            int accountId = Integer.parseInt(accountIdStr);
```

Comment:
add try catch here and then handle exception downwards. sometimes this accountIdStr gives null

Reply:
There is a try catch for the entire job. accountIdStr is coming from the keyset. In case the keyset itself is null, gets handled here. In case the keyset contains something like this [ 1234, 2345, null, 3456 ] , in this case the logic would break. should we add a try catch inside the for loop as well, didn't seem necessary since there was one outside ?